nodes/stampzilla-server/store/devices.go

```diff
@@ -10,6 +10,10 @@ func (store *Store) GetDevices() *devices.List {
 
 func (store *Store) AddOrUpdateDevice(dev *devices.Device) {
 	store.Devices.Add(dev)
+	node := store.GetNode(dev.ID.Node)
+	if a, ok := node.Aliases[dev.ID]; ok {
```

Comment:
Wou are reading node.Aliases[dev.ID] with no locks here? Could collide with `node.Aliases[device.ID] = device.Alias`which has locks...

Reply:
Those locks gets messy... perhaps have a single goroutine responsible for certain type?